content/rdi/quickstart/write-behind-guide.md

```diff
@@ -10,15 +10,17 @@ draft:
 hidden: false
 ---
 
-This guide takes you through the creation of a write-behind pipeline.
+This guide takes you through the creation of a write-behind and read-through pipelines.
```

Comment:
remove `a`:
his guide takes you through the creation of write-behind and read-through pipelines.

Reply:
done